examples/hashes/connector.py

```diff
@@ -0,0 +1,115 @@
+# This example shows how to calculate hash of fields to be used as primary key. This is useful in scenarios where the
+# incoming rows do not have any field suitable to be used as a Primary Key.
+# Fivetran recommends to define primary keys (https://fivetran.com/docs/connectors/connector-sdk/best-practices#declaringprimarykeys) when writing a connector.
+# This is to avoid disruptions caused due to a schema change (addition/removal of a column, etc) that might be needed in the maintainance phase of your connector.
```

Comment:
```suggestion
# This is to avoid disruptions caused due to a schema change (addition/removal of a column, etc) that might be needed in the maintenance phase of your connector.
```